environment.yml

```diff
@@ -36,11 +35,22 @@ dependencies:
   - sphinxcontrib-bibtex
   - sphinx_rtd_theme
   - threadpoolctl
-  - tabulate
-  - widgetsnbextension
+  - tabulate>=0.8.7
+  - widgetsnbextension==3.5.1
+  - Jinja2==2.11
```

Comment:
Same comment as before, are these pins actually necessary? (we unpinned jinja2 a couple of PRs up), if so we need to work out why and what path is needed for us to unpin. Keeping things pinned is unfortunately too unssustainable and I'd like to make sure we can avoid it as much as possible.

Reply:
Sorry, unpinned now.